runewords.json

```diff
@@ -1,80 +1,2006 @@
-    "Chaos": "Fal + Ohm + Um (Claws)",
-    "Crescent Moon": "Shael + Um + Tir (Axes/Swords/Polearms)",
-    "Delirium": "Lem + Ist + Io (Helms)",
-    "Duress": "Shael + Um + Thul (Body Armor)",
-    "Enigma": "Jah + Ith + Ber (Body Armor)",
-    "Eternity": "Amn + Ber + Ist + Sol + Sur (Melee Weapons)",
-    "Exile": "Fal + Ohm + Ort + Jah (Axes/Hammers)",
-    "Gloom": "Fal + Um + Pul (Body Armor)",
-    "Hand of Justice": "Sur + Cham + Amn + Lo (Weapons)",
-    "Heart of the Oak": "Ko + Vex + Pul + Thul (Staves/Maces)",
-    "Doom": "Hel + Ohm + Um + Lo + Cham (Axes/Polearms/Hammers)",
-    "Kingslayer": "Mal + Um + Gul + Fal (Swords/Axes)",
-    "Passion": "Dol + Ort + Eld + Lem (Weapons)",
-    "Prudence": "Mal + Tir (Body Armor)",
-    "Sanctuary": "Ko + Ko + Ma (Shields)",
-    "Splendor": "Eth + Lum (Shields)",
-    "Stone": "Shael + Um + Pul + Lum (Body Armor)",
-    "Wind": "Sur + El (Melee Weapons)",
-    "Brand": "Jah + Lo + Mal + Gul (Missile Weapons, Ladder)",
-    "Death": "Hel + El + Vex + Ort + Gul (Swords/Axes, Ladder)",
-    "Destruction": "Vex + Lo + Ber + Jah + Ko (Polearms/Swords, Ladder)",
-    "Dragon": "Sur + Lo + Sol (Body Armor/Shields, Ladder)",
-    "Dream": "Io + Jah + Pul (Helms/Shields, Ladder)",
-    "Edge": "Tir + Tal + Amn (Missile Weapons, Ladder)",
-    "Faith": "Ohm + Jah + Lem + Eld (Missile Weapons, Ladder)",
-    "Fortitude": "El + Sol + Dol + Lo (Weapons/Body Armor, Ladder)",
-    "Grief": "Eth + Tir + Lo + Mal + Ral (Swords/Axes, Ladder)",
-    "Plague": "Cham + Fal + Um (Weapons, Disabled)",
-    "Harmony": "Tir + Ith + Sol + Ko (Missile Weapons, Ladder)",
-    "Ice": "Amn + Shael + Jah + Lo (Missile Weapons, Ladder)",
-    "Infinity": "Ber + Mal + Ber + Ist (Polearms, Ladder)",
-    "Insight": "Ral + Tir + Tal + Sol (Polearms/Staves, Ladder)",
-    "Last Wish": "Jah + Mal + Jah + Sur + Jah + Ber (Swords/Hammers/Axes, Ladder)",
-    "Lawbringer": "Amn + Lem + Ko (Swords/Hammers/Scepters, Ladder)",
-    "Oath": "Eth + Tir + Lo + Mal + Ral (Swords/Axes/Maces, Ladder)",
-    "Obedience": "Hel + Ko + Thul + Eth + Fal (Polearms, Ladder)",
-    "Phoenix": "Vex + Vex + Lo + Jah (Weapons/Shields, Ladder)",
-    "Pride": "Cham + Sur + Io + Lo (Polearms, Ladder)",
-    "Rift": "Hel + Ko + Lem + Gul (Polearms/Scepters, Ladder)",
-    "Spirit": "Tal + Thul + Ort + Amn (Swords/Shields, Ladder)",
-    "Voice of Reason": "Lem + Ko + El + Eld (Swords/Maces, Ladder)",
-    "Wrath": "Pul + Lum + Ber + Mal (Missile Weapons, Ladder)",
-    "Bone": "Sol + Um + Um (Body Armor)",
-    "Enlightenment": "Pul + Ral + Sol (Body Armor)",
-    "Myth": "Hel + Amn + Nef (Body Armor)",
-    "Peace": "Shael + Thul + Amn (Body Armor)",
-    "Principle": "Ral + Gul + Eld (Body Armor)",
-    "Rain": "Ort + Mal + Ith (Body Armor)",
-    "Treachery": "Shael + Thul + Lem (Body Armor)"
-}
+[
+    {
+        "name": "Ancient's Pledge",
+        "allowedItems": [
+            "3 Soc Shields"
```

Comment:
Maybe we should go without the "X Soc" parts in `allowedItems`? This piece of data is redundant as it can be generated when needed based on the amount of runes.

Let me know your thoughts, I will gladly merge this either way.

Reply:
My thought was using this as a way to split outputs for runewords like phoenix, dream etc where you have different types of items with the same runeword and different outcomes.

Example: Spirit
```JS
"allowedItems": [
    "4 Soc Swords",
    "4 Soc Shields"
],
"properties": [
    "4 Soc Swords",
    "+2 To All Skills",
    "+25-35% FCR",
    "+55% FHR",
    "Adds 1-50 Lightning Dmg",
    "Adds 3-14 Cold Dmg 3 Second Duration (Normal)",
    "+75 Poison Dmg Over 5 Seconds",
    "7% LL Per Hit",
    "+250 Def Vs. Missile",
    "+22 To Vit",
    "+89-112 To Mana",
    "+3-8 Magic Absorb",
    "4 Soc Shields",
    "+2 To All Skills",
    "+25-35% FCR",
    "+55% FHR",
    "+250 Def Vs. Missile",
    "+22 To Vit",
    "+89-112 To Mana",
    "Cold Res +35%",
    "Lightning Res +35%",
    "Poison Res +35%",
    "+3-8 Magic Absorb",
    "Attacker Takes Dmg of 14"
  ],
```

My idea is to when I have more than 1 allowed item, I expect one of the properties to be that allowed item entry (as a way to split the outputs).

I can also create 2 entries, spirit shield and spirit armor.